defaults/main.yml

```diff
@@ -26,3 +26,5 @@ monasca_log_level: WARN
 zookeeper_connection_timeout_ms: 1000000
 
 verify: True
+opt_dir: "/opt"
+download_tmp_dir: "{{ opt_dir }}/.download_dir"
```

Comment:
This is effectively a rename of kafka_tarball_location I don't see why you want to do that but it is okay if you clean up the previous value. One thing I don't like is the default being a hidden directory, it makes it difficult to find and occasionally it is nice to find the original source.